impeller/entity/contents/filters/filter_contents.h

```diff
@@ -178,11 +184,28 @@ class FilterContents : public Contents {
   virtual void SetRenderingMode(Entity::RenderingMode rendering_mode);
 
  private:
+  /// @brief  Internal utility method for |GetLocalCoverage|.
```

Comment:
The brief should explain what it does, not how it is used.  It should be something like "Computes the filter's coverage sans coverage hint.

See also:
  - |GetLocalCoverage| - This combines this result with the coverage hint.
"

Reply:
I basically condensed these comments (including their non-brief parts) into just a single brief sentence.